chain/index/chain-indexing-overview-for-rpc-providers.MD

```diff
@@ -60,34 +71,137 @@ The ChainIndexer [periodically runs](https://github.com/filecoin-project/lotus/b
 [Fevm]
 EthTxHashMappingLifetimeDays = 0
 
+TODO: https://github.com/filecoin-project/lotus/pull/12450#discussion_r1792743250
+> see note in #12421, this one is duplicated with a moved redirect
+
 [Events]
 DisableHistoricFilterAPI = false
 DatabasePath = ""
 ```
 
-## Upgrade Steps
+## Upgrade
+
+### Preparation
+> **Note:** One can upgrade/downgrade between [pre-ChainIndexer](#previous-indexing-system) and [with-ChainIndexer](#chainindexer-indexing-system) Lotus versions without conflict because they persist state to different directories and don't rely on each other. No backup is necessary (but extra backups don't hurt). There is still a [backfilling step though when downgrading](#downgrade-steps).
+
+These upgrade steps assume one has multiple nodes in their fleet and can afford to have a node not handling traffic potentially for days potentially per [backfill timing below](#backfill-timing).
```

Comment:
```suggestion
These upgrade steps assume one has multiple nodes in their fleet and can afford to have a node not handling traffic, potentially for days per [backfill timing below](#backfill-timing).
```

Reply:
Do we want to differentiate between "archival level" nodes and splitstore nodes here? because a case like api.node.glif.io is very different to the archival case.

See a discussion with glif in this thread which is very relevant to this section: https://github.com/filecoin-project/lotus/pull/12421#discussion_r1798751016

Reply:
I attempted this in https://github.com/filecoin-project/lotus/pull/12600/commits/06acfdaaa8d45b0c10eddabe6fb5d276148f7f88